### DISCUSSION TITLE
New release for a new year!

### DISCUSSION THREAD

**Discussion by gbishop** [UNC-Project-Open-AAC/OS-DPI#236] [Announcements]
We passed 1000 commits and over 12,600 lines of code!

This is a huge reorganization of the site and the code.

1. The top level is much cleaner now that the code has moved down into the src folder.
2. Component properties now accept Excel like formulas enabling dynamic behavior that was not previously possible.
3. The Monitor now scrolls and shows undefined values in red.
4. The tabs in the Designer now scroll below the tab buttons.
5. The internal code has been refactored and cleaned up.
6. Only one Method may be Active at a time. Multiple active methods has been a source of continuing confusion. This is a **breaking change**.
7. Many small cosmetic changes that I hope improve usability.

As with any change I may have introduced some bugs. Please report any issues immediately.

**Comment by aetkie** [comment]
What an achievement!
